packages/react-data-grid/src/common/types.ts

```diff
@@ -120,17 +124,17 @@ export interface Editor<TValue = never> extends React.Component {
   readonly disableContainerStyles?: boolean;
 }
 
-export interface FormatterProps<TValue, TDependentValue = unknown, TRow = any> {
+export interface FormatterProps<TValue, TDependentValue = any, TRow = any> {
```

Comment:
I put `any` so we can create generic formatters. `unknown` might work, I will have to check